protocols/virtualfund/virtualfund.go

```diff
@@ -92,8 +92,8 @@ func (c *Connection) handleProposal(sp consensus_channel.SignedProposal) error {
 	return nil
 }
 
-// Funded computes whether the ledger channel on the receiver funds the guarantee expected by this connection
-func (c *Connection) Funded() bool {
+// IsFundingTheTarget computes whether the ledger channel on the receiver funds the guarantee expected by this connection
+func (c *Connection) IsFundingTheTarget() bool {
 	g := c.getExpectedGuarantee()
 	return c.Channel.Includes(g)
 }
```

Comment:
How about  `includesExpectedGuarantee`?

Reply:
I'm going to leave this as is because it:
- is a more "active voice"
- asks less of the reader: someone can tell that the connection has "done its job" with respect to the virtualfund objective without knowing anything about the internals of a ledger channel (including what a guarantee is, for example)